session.go

```diff
@@ -274,6 +288,10 @@ func (s *Session) RunUpdate(stmt spanner.Statement) (int64, error) {
 	opts := spanner.QueryOptions{
 		Priority: s.currentPriority(),
 	}
+	if s.tc.transactionTag != "" {
```

Comment:
Same to the above. We don't need this if guard.

Reply:
fixed.